dev-tools/package.yaml

```diff
@@ -198,10 +198,6 @@ executables:
       - kore
       - optparse-applicative
       - transformers
-    ghc-options:
-    - -rtsopts
-    - -threaded
-    - -with-rtsopts=-N
   kore-parser:
```

Comment:
I would always leave the `-rtsopts` active (without it, it is not allowed to pass `+RTS` options at all)
But they can of course be added quickly in a local build if need be...

Reply:
Sounds good, I've brought them back.